### ISSUE TITLE
"Audio cue" no longer can be found in command palette / settings

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#211049]
We have renamed "Audio Cues" to "Signal Announcements" - which is fine. The challange is that we already have a lot of users that are familiar with the term "audio cue". So imagine a user searching for "audio cue" in the command palette or settings and not finding anything.

We should add "audio" and "cue" to keywords to make sure that the commands and settings are picked up when users search for this.

In the future we should be careful when re-naming user facing features.

**Comment by isidorn** [comment]
Not in settings


![Image](https://github.com/microsoft/vscode/assets/1926584/9368c9e0-e131-494a-b416-52f405774ccc)
![Image](https://github.com/microsoft/vscode/assets/1926584/131d43b9-1d81-4431-b5ab-3fdb5949f264)

**Comment by meganrogge** [comment]
When verifying, make sure that searching for `alert, announcement, sound, audio cue, or signal` all show the accessibility signal results
